core/bind/core_bind.cpp

```diff
@@ -2367,6 +2367,11 @@ String _Directory::get_current_dir() {
 	ERR_FAIL_COND_V_MSG(!d, "", "Directory must be opened before use.");
 	return d->get_current_dir();
 }
+String _Directory::get_current_dir_without_drive() {
```

Comment:
Maybe we could simply add an "include_drive" as an optional argument to get_current_dir(). Those two functions do the same thing in general, so that might be cleaner.

Reply:
That has a point, but I still prefer a separate method. Boolean parameters are hard to read in the calling location.

Reply:
I kind of disagree, since most of the time the parameter is not there it does not make a significant difference IMHO. It's better than adding functions with longs name for each possible case.
But well, it's not that important. If others have an opinion on that that would be nice :)

Reply:
I also prefer optional arguments to multiple functions. I had this discussion earlier in the context of the C# string code. Should be `get_current_dir(bool include_drive = true)` (or false)

P.S. This needs empty lines as per #33027 "Enforce one line between function implementations"

Reply:
Change done.

Regarding empty lines, such a cosmetic change is something to be done aside of this PR, especially after the requested changes, since now it's not even adding/removing methods at all.